packages/cli-tools/src/startServerInNewWindow.ts

```diff
@@ -1,18 +1,22 @@
 import path from 'path';
 import fs from 'fs';
 import execa from 'execa';
-import {
-  CLIError,
-  logger,
-  resolveNodeModuleDir,
-} from '@react-native-community/cli-tools';
+import {CLIError} from './errors';
+import resolveNodeModuleDir from './resolveNodeModuleDir';
+import logger from './logger';
 
-export function startServerInNewWindow(
+function startServerInNewWindow(
   port: number,
-  terminal: string,
   projectRoot: string,
   reactNativePath: string,
+  terminal?: string,
 ) {
+  if (!terminal) {
```

Comment:
This check is preventing android build on windows. On windows - so far we did not need any `terminal` value. It always used to fall back to default `cmd.exe` ( line no 106 )

In this case I think the proper fix should be fixing the `getDefaultUserTerminal` which currently doesn't have any default terminal for windows.

Reply:
Or only apply this check on platforms other than windows.